app/assets/javascripts/initializers/initializeDrawerSliders.js

```diff
@@ -1,57 +1,50 @@
 function initializeDrawerSliders() {
-  if(!initializeSwipeGestures.called) {
-    swipeState = "middle";
+  if (!initializeSwipeGestures.called) {
+    swipeState = 'middle';
     initializeSwipeGestures();
   }
-  if (document.getElementById("on-page-nav-controls")){
-    if (document.getElementById("sidebar-bg-left")){
-      document.getElementById("sidebar-bg-left").onclick = function(){
-        swipeState = "middle";
-        slideSidebar("left","outOfView");
-      }
-    }
-    if (document.getElementById("sidebar-bg-right")){
-      document.getElementById("sidebar-bg-right").onclick = function(){
-        swipeState = "middle";
-        slideSidebar("right","outOfView");
-      }
-    }
 
-    if (document.getElementById("on-page-nav-butt-left")){
-      document.getElementById("on-page-nav-butt-left").onclick = function(){
-        swipeState = "left"
-        slideSidebar("left","intoView");
-      }
-    }
-    if (document.getElementById("on-page-nav-butt-right")){
-      document.getElementById("on-page-nav-butt-right").onclick = function(){
-        swipeState = "right"
-        slideSidebar("right","intoView");
+  if (document.getElementById('on-page-nav-controls')) {
+    const drawerSliders = [
+      { selector: 'sidebar-bg-left', swipeState: 'middle', side: 'left', view: 'outOfView', },
+      { selector: 'sidebar-bg-right', swipeState: 'middle', side: 'right', view: 'outOfView', },
+      { selector: 'on-page-nav-butt-left', swipeState: 'left', side: 'left', view: 'intoView', },
+      { selector: 'on-page-nav-butt-right', swipeState: 'right', side: 'right', view: 'intoView', },
+    ];
+
+    drawerSliders.forEach(drawerSliders => {
+      const element = document.getElementById(drawerSlider.selector);
+      if (element) {
+        element.onclick = function() {
+          swipeState = drawerSliders.swipeState;
+          slideSidebar(drawerSliders.side, drawerSliders.view);
+        };
       }
-    }
+    });
+
     InstantClick.on('change', function() {
       document.getElementsByTagName('body')[0].classList.remove('modal-open');
-      slideSidebar("right","outOfView");
-      slideSidebar("left","outOfView");
+      slideSidebar('right', 'outOfView');
+      slideSidebar('left', 'outOfView');
     });
     listenForNarrowMenuClick();
   }
 }
 
 function listenForNarrowMenuClick(event) {
-  var navLinks = document.getElementsByClassName("narrow-nav-menu");
-  var narrowFeedButt = document.getElementById("narrow-feed-butt");
-  for (var i = 0; i < navLinks.length; i++) {
-    document.getElementById("narrow-nav-menu").classList.remove("showing");
+  var navLinks = document.getElementsByClassName('narrow-nav-menu');
```

Comment:
If you have time, it'd be nice to migrate `var` to `const` and `let` where necessary. I use `const` as a default, but there are no strong opinions on this in the codebase.